src/__tests__/index.spec.ts

```diff
@@ -48,6 +48,34 @@ const B = /*#__PURE__*/String("b");`,
   title: "Annotated #__PURE__ for fn(fn())",
   src: `export const A = fn(fn())`,
   dest: `export const A = /*#__PURE__*/fn( /*#__PURE__*/fn());`,
+}, {
+  title: "Annotate only top level calls",
+  src: `export const A = () => { var B = call(); };`,
+  dest: `export const A = () => {
+  var B = call();
```

Comment:
`var B = call();` should be removed if it is not be used.

Reply:
the pure call annotation is fine here

Reply:
this is a job for a linter https://eslint.org/docs/rules/no-unused-vars

`/*#__PURE__*/` got implemented to indicate that it is safe to dead code eliminate scenarios like:

```js
export const add = curry((a, b) => a + b)
export const increment = add(1)
```

While `increment` is technically used now (because of the `export` statement) if we do not import it (as consumers of the package) becomes unused but cannot be completely dead code eliminated because of the function call.

Reply:
Not everyone used a linter